docker-compose.yml

```diff
@@ -16,36 +24,72 @@ services:
   notifier:
     depends_on:
       - nats
+    container_name: meditor_notifier
+    build: packages/notifier
+    restart: on-failure
+    env_file:
+      - .env
+    volumes:
+      - ./packages/notifier:/usr/src/app
+      - /usr/src/app/node_modules
 
   app:
-    depends_on:
-      - legacy-api
+    container_name: meditor_app
+    build: packages/app
+    restart: on-failure
+    env_file:
+      - .env
+    ports:
+      - "127.0.0.1:4000:4000"
+    develop:
```

Comment:
New develop section for the app, makes volume mounting unnecessary. If you do `docker-compose up --watch` or `docker compose watch` these take effect